components/header/index.css

```diff
@@ -19,7 +19,7 @@
   flex-grow: 0;
 }
 
-.menu-toggle {
+.header-area-brand .menu-toggle {
```

Comment:
Are you experiencing specificity issues?

I guess this file conflicts with #25.

Reply:
Yes, there will be some conflicts. Let's merge your PR first and I will resolve conflicts here, ok?

Reply:
Let's do so!